SConstruct

```diff
@@ -1249,6 +1228,39 @@ for library in ( "GafferUI", ) :
 	if int( env["QT_VERSION"] ) > 4 :
 		addQtLibrary( library, "Widgets" )
 
+# Add required libraries for Windows
+
+if env["PLATFORM"] == "win32" :
+
+	for library in ( "Gaffer", ) :
+
+		if "envAppends" not in libraries[library] :
+			libraries[library]["envAppends"] = {}
```

Comment:
This could maybe be written a bit more succinctly like so :

```
envAppends = libraries[library].getdefault( "envAppends", {} )
envAppends["LIBS"] = ...
```

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/4524/commits/acdd11677688af02288e4ccef9b876fb68793f28

Reply:
And just to make sure I'm not missing something, should `getdefault` in your snippet be `setdefault`?

Reply:
Yep - that's right, sorry for the mistake...